BlueprintUI/Sources/Animation/Transition.swift

```diff
@@ -0,0 +1,84 @@
+import UIKit
+
+
+/**
+ Wraps a content element and adds transitions when the element appears, disappears, or changes layout.
+ */
+public struct Transition<Wrapped:Element> : Element {
```

Comment:
More generally... How do we feel about moving towards making `wrapped` generic? Nice for instantiation and then configuration, eg:

```
var transition = Transition(onAppear: ..., wrapping: foo)

transition.wrapping.thing = bar
```

Reply:
Are they? Generics are pretty table stakes in Swift – and this doesn't affect the ability to use the value as an `Element`. If it were up to me, we'd be going back through all elements and making them generic around `Element`.

Reply:
Going to remove this for now, but I think it makes sense to convert these to generics eventually.